packages/format-currency/src/index.js

```diff
@@ -9,9 +7,11 @@ import { numberFormat } from 'i18n-calypso';
  * Internal dependencies
  */
 import { getCurrencyDefaults } from './currencies';
-
 export { getCurrencyDefaults };
 
+import { CURRENCIES } from './currencies';
+export { CURRENCIES };
```

Comment:
Since `CURRENCIES` isn't used otherwise in this file, you can contract this to

```js
export { CURRENCIES } from './currencies';
```

Reply:
```suggestion
```